### PR TITLE
Replace exception message by custom error message when uploading avatar with invalid extension

### LINKED ISSUE #28484: BO - Exception thrown when adding invalid avatar format in profile page
### Prerequisites

- [X] I understand and accept the project's [code of conduct](https://github.com/PrestaShop/PrestaShop/blob/develop/CODE_OF_CONDUCT.md)
- [X] I have already [searched in existing issues](https://github.com/PrestaShop/PrestaShop/issues?q=is%3Aissue+label%3ABug) and found no previous report of this bug.

### Describe the bug and add screenshots

Exception thrown when adding invalid avatar format in profile page
PS 1.7.8.0

https://user-images.githubusercontent.com/89604350/168235398-6705cede-7d87-4f6c-8bd1-849be6d6b595.mp4

PS1.7.7.8
image

### Expected behavior

_No response_

### Steps to reproduce

1. Go to BO > Advanced parameters > Team > Profiles tab
2. Fill the name 
3. Choose a file with a format different from .gif, .jpg, .png
4. see error

### PrestaShop version(s) where the bug happened

1.7.8.5/1.7.8.0

### PHP version(s) where the bug happened

7.4

### If your bug is related to a module, specify its name and its version

_No response_

### PR DESCRIPTION
**Pull request by lartist**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | 1.7.8.x
| Description?      | replace exception message by custom error message when uploading avatar with invalid extension
| Type?             | bug fix 
| Category?         | BO 
| BC breaks?        |  no
| Deprecations?     |no
| Fixed ticket?     | Fixes #28484
| Related PRs       | 
| How to test?      | https://github.com/PrestaShop/PrestaShop/issues/28484
| Possible impacts? |

### CODE REVIEW

File: src/PrestaShopBundle/Controller/Admin/Configure/AdvancedParameters/ProfileController.php [hunk 1]
```diff
@@ -286,6 +287,10 @@ public function bulkDeleteAction(Request $request)
     protected function getErrorMessages()
     {
         return [
+            UploadedImageConstraintException::class => $this->trans(
+                'Image format not recognized, allowed formats are: %s',
+                'Admin.Notifications.Error'
+            ),
```
Comment (PululuK) [inline src/PrestaShopBundle/Controller/Admin/Configure/AdvancedParameters/ProfileController.php]: Hello @lartist  :)
Missing allowed formats printf params. You can use  `ImageManager::EXTENSIONS_SUPPORTED`.

```php
UploadedImageConstraintException::class => $this->trans(
    'Image format not recognized, allowed formats are: %s',
    'Admin.Notifications.Error',
    [
        implode(', ', ImageManager::EXTENSIONS_SUPPORTED),
    ]
),
```

See `PrestaShopBundle\Controller\Admin\ProductImageController::uploadImageAction` for example

**Comment by matks** [comment]
@lartist No need to add "fix(BO)" as you also mention the category in the PR content. Just keep your PR title, it will be added to the Changelog like this https://github.com/PrestaShop/PrestaShop/blob/develop/docs/CHANGELOG.txt

File: src/PrestaShopBundle/Controller/Admin/Configure/AdvancedParameters/ProfileController.php (hunk 1 above)
Comment (matks) [reply src/PrestaShopBundle/Controller/Admin/Configure/AdvancedParameters/ProfileController.php]: Nice suggestion 👍

**Review by matks — COMMENTED** [review]
Code is good, please just add @PululuK suggestion 🎉

**Review by jolelievre — DISMISSED** [review]
Thanks @lartist

**Review by l-delin — DISMISSED** [review]

**Review by jolelievre — DISMISSED** [review]
Thanks @lartist

**Review by matks — DISMISSED** [review]

**Review by florine2623 — COMMENTED** [review]
Hello @lartist ,

It is NOK:
Screenshot 2022-10-07 at 16 06 23

There should not be the "image/" before the extension.

Could you check?
Thanks!

**Comment by Progi1984** [comment]
@florine2623 It's not extensions. It is mime types.

**Comment by florine2623** [comment]
Thanks @Progi1984 ,

Then it is QA ✅ !

**Comment by Progi1984** [comment]
@florine2623 No, errors should contain extensions and not image types.

FYI @lartist

**Comment by lartist** [comment]
> 

Hi @Progi1984,

In the file ImageManager in the version 1.7.8.x there is only 
    const MIME_TYPE_SUPPORTED = [
        'image/gif',
        'image/jpg',
        'image/jpeg',
        'image/pjpeg',
        'image/png',
        'image/x-png',
    ];

It seems like the following const was removed by error (https://github.com/PrestaShop/PrestaShop/commit/cd4039defd1d5a2d1c11a82b40b04736e1e10a5b) and added back to the version 8. Should I have to add it in 1.7.8.x ?

   const EXTENSIONS_SUPPORTED = [
        'gif',
        'jpg',
        'jpeg',
        'jpe',
        'png',
        'webp',
    ];

**Comment by kpodemski** [comment]
@lartist makes sense, remember to remove webp on 1.7 branch as it doesn't support it :)

**Comment by lartist** [comment]
Done

**Review by matks — APPROVED** [review]

**Review by FabienPapet — APPROVED** [review]

**Review by kpodemski — APPROVED** [review]

**Review by florine2623 — APPROVED** [review]
Hello all,

Now the error message is the following:
Screenshot 2022-10-17 at 11 36 52

It is QA ✅ 
Thanks!

**Comment by jolelievre** [comment]
Thanks @lartist and @florine2623

**Pull request merged**